doc/doc/basics/extractors.md

```diff
@@ -20,7 +20,12 @@ class contains different extractor *styles*:
   previous systems, but with limited performance
   - [`tsv_extractor`](#tsv_extractor): moderate flexibility and performance
   - [`plpy_extractor`](#plpy_extractor): parallel database-built-in extractors
-  with restricted flexibility
+  with seriously restricted flexibility because only the UDF function definition
+  is shipped to the DB
+  - [`piggy_extractor`](#piggy_extractor): a distributed language-agnostic pipe-based extractor driver.
+  We ship the entire runtime environment (code, data, and pip libraries) to
+  the DB server(s) and avoid multi-user collision by staging separate environments
+  for concurrent users. See `examples/spouse_example/piggy_extractor/`.
```

Comment:
Could you add more detail about using piggy?  There's no `#piggy_extractor` section defined here.  For example, what extra config is needed, how piggy knows what to ship, which databases are supported, etc.  It's probably a bad idea to have users stare at the piggy example and expect them to figure out what's changed from tsv or other examples.